entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# Remove a potentially pre-existing server.pid for Rails.
+rm -f /app/tmp/pids/server.pid
+
+# Then exec the container's main process (what's set as CMD in the Dockerfile).
```

Comment:
This is a little inaccurate, as `CMD` only sets a default command that can be overwritten (e.g. via command line parameters or `docker-compose`)…

Reply:
:thinking: err, yes? That's what we want, right?